concreate/builders/docker.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import subprocess
+import os
+
+from concreate.builder import Builder
+from concreate.errors import ConcreateError
+
+
+logger = logging.getLogger('concreate')
+
+
+class DockerBuilder(Builder):
+    """This class wraps docker build command to build and image"""
+
+    def build(self, build_args):
+        """After the source siles are generated, the container image can be built.
+        We're using Docker to build the image currently.
+
+        This can be changed by specifying the tags in CLI using --build-tags option.
+
+        Args:
+          build_tags - a list of image tags
+        """
+        tags = build_args.build_tags
+        cmd = ["docker", "build"]
+
+        # Custom tags for the container image
+        logger.debug("Custom tags for container image specified: %s" %
```

Comment:
This should be reworded. After your refactoring these can be default or custom tags.